frontend/src/features/mapSideBar/tabs/ltsa/LtsaTabView.tsx

```diff
@@ -22,22 +22,35 @@ export interface ILtsaTabViewProps {
   ltsaData?: LtsaOrders;
   ltsaRequestedOn?: Date;
   loading: boolean;
+  pid?: string;
 }
 
 export const LtsaTabView: React.FunctionComponent<ILtsaTabViewProps> = ({
   ltsaData,
   ltsaRequestedOn,
   loading,
+  pid,
 }) => {
   const titleNameSpace = 'titleOrders.0.orderedProduct.fieldedData';
 
   return (
     <>
       <LoadingBackdrop show={loading} parentScreen={true} />
-      {!loading && !ltsaData ? (
+
+      {!pid ? (
+        <FormSection>
+          <b>
+            This property does not have a valid PID.
+            <br />
+            <br /> Only properties that are associated to a valid PID can display corresponding data
+            from LTSA.
+          </b>
+        </FormSection>
+      ) : !loading && !ltsaData ? (
```

Comment:
Instead of having both on the same markup it could be in two different returns.
```
if(notValidLtsa)
{
   return <>Sorry, no ltsdataa</>
}
else
{
   return <>...Stuff that goes with ltsa</>
}
```